build.psm1

```diff
@@ -1838,84 +1859,6 @@ function Start-DevPowerShell {
     }
 }
 
-
-<#
-.EXAMPLE
-PS C:> Copy-MappedFiles -PslMonadRoot .\src\monad
-
-copy files FROM .\src\monad (old location of submodule) TO src/<project> folders
-#>
-function Copy-MappedFiles {
```

Comment:
Sure thing, Actually @vors submitted a PR to do so: #3407. Maybe we need committee's approval to go ahead with that PR?